### PR TITLE
Add acorn option and nested objects from dataset.

### PR DESCRIPTION
**Pull request by jfrancoa**
This commit adds support for enabling acorn on collection creation, as well as during the collection update.

Also, it includes couple of new properties of type object by adding the property type and improving the import_json function to handle properties with camelCase naming

### CODE REVIEW

File: weaviate_cli/managers/data_manager.py [hunk 1]
```diff
@@ -48,7 +53,7 @@ def __import_json(
                         for prop in properties:
                             prop_name = PROPERTY_NAME_MAPPING.get(prop.name, prop.name)
                             if prop_name in obj:
-                                added_obj[prop_name] = self.__convert_property_value(
+                                added_obj[prop.name] = self.__convert_property_value(
```
Comment (rlmanrique) [inline weaviate_cli/managers/data_manager.py:56]: typo here? Shouldn't it be prop_name?

File: weaviate_cli/managers/data_manager.py [hunk 2]
```diff
@@ -95,6 +100,18 @@ def create_single_object() -> Dict:
             date = datetime.strptime("1980-01-01", "%Y-%m-%d")
             random_date = date + timedelta(days=random.randint(1, 15_000))
             release_date = random_date.strftime("%Y-%m-%dT%H:%M:%SZ")
+            # production_companies = [
+            #     {"name": get_random_string(3), "id": random.randint(1, 1000)}
+            #     for _ in range(random.randint(1, 3))
+            # ]
```
Comment (rlmanrique) [inline weaviate_cli/managers/data_manager.py:106]: Delete commented lines?

**Review by rlmanrique — CHANGES_REQUESTED** [review]

File: weaviate_cli/managers/data_manager.py (hunk 1 above)
Comment (jfrancoa) [reply weaviate_cli/managers/data_manager.py:56]: No, it's on purpose. Here prop is the property obtained from the collection [properties configuration](https://github.com/weaviate/weaviate-cli/blob/main/weaviate_cli/managers/collection_manager.py#L120-L134). Then, to be able to use the contextionary vectorizer (it requires the properties names in camelCase format, it won't accept a string separated by underscore) we have the PROPERTY_NAME_MAPPING that maps the property from camelCase to the underscore one.
Therefore, when we add the object we need to use the prop.name (which is the same name used when defining the properties in class creation). As it was before, it was skipping all properties which had camelCase and it wouldn't add any object for those, that's why I swapped also the content in PROPERTY_NAME_MAPPING.

File: weaviate_cli/managers/data_manager.py (hunk 2 above)
Comment (jfrancoa) [reply weaviate_cli/managers/data_manager.py:106]: Oh yes, thanks!

**Review by rlmanrique — APPROVED** [review]

**Pull request merged**